packages/starknet-snap/test/src/switchNetwork.test.ts

```diff
@@ -56,7 +52,6 @@ describe('Test function: switchNetwork', function () {
     expect(stateStub).to.be.calledOnce;
     expect(dialogStub).to.be.calledOnce;
     expect(state.currentNetwork).to.be.eql(STARKNET_MAINNET_NETWORK);
-    expect(state.networks.length).to.be.eql(3);
   });
 
   it('should skip autherize when enableAutherize is false or omit', async function () {
```

Comment:
same here `authorize` and `enableAuthorize` instead of `autherize` and `enableAutherize`

Reply:
lets do it in other branch, this branch is remove network